Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Connected Animations/Pages/ThirdPage.xaml.cs

```diff
@@ -2,24 +2,39 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.Toolkit.Uwp.SampleApp.Data;
 using Microsoft.Toolkit.Uwp.SampleApp.Models;
+using System;
+using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
 using Windows.UI.Xaml.Navigation;
 
 namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages.ConnectedAnimations.Pages
 {
     public sealed partial class ThirdPage : Page
     {
-        private Item item;
+        private PhotoDataItem item;
 
         public ThirdPage()
         {
+            Loaded += OnLoaded;
             this.InitializeComponent();
         }
 
+        private void OnLoaded(object sender, RoutedEventArgs e)
```

Comment:
These methods are not used and don't bring a lot of values. You should remove them